analyses/MN_cd_2010/doc_MN_cd_2010.md

```diff
@@ -0,0 +1,28 @@
+# 2010 Minnesota Congressional Districts
+
+## Redistricting requirements
+In Minnesota, districts must:
+
+1. be contiguous
+2, have equal populations
+3. comply with VRA section 2
+4. be geographically compact
+5. preserve political subdivisions and communities of interest as possible 
+6. avoid pairing incumbents but also cannot give unfair advantage to incumbents (least important criteria)
+
+https://www.mncourts.gov/mncourtsgov/media/CIOMediaLibrary/2011Redistricting/A110152Order11-4-11.pdf
+
+### Interpretation of requirements
+We do not adhere to all criteria in the guidelines. We include the following constraints:
+
+1. We enforce a maximum population deviation of 0.5%. 
+2. We use a pseudo-county constraint to help preserve county and municipality boundaries.
+
+## Data Sources
+Data for Minnesota comes from the ALARM Project's [2010 Redistricting Data Files](https://alarm-redist.github.io/posts/2021-08-10-census-2020/).
+
+## Pre-processing Notes
+No manual pre-processing decisions were necessary.
+
+## Simulation Notes
+We sample 5,000 districting plans  for Minnesota across two independent runs of the SMC algorithm.  We use a pseudo-county constraint to limit the county and municipality splits. Municipality lines used are in , which are all counties with populations larger than 60% the target population for district.
```

Comment:
maybe add a tiny bit more detail here (see some other state PRs for examples) to explain the pseudocounty constraint.

As far as getting the counties themselves, the numbers you get out of the R code when you run it like `037` are county FIPS codes.  IF you look up "MN county 037" you'll see the name that corresponds to the code.  These are also stored in `tigris::fips_codes` if you want a programmatic solution